### ISSUE TITLE
Fix Kobo Mark7 mxcfb handling

### ISSUE THREAD

**Issue by NiLuJe** [koreader/koreader-base#694]
Creating a new issue, as a followup of #390 ;).

The Mark7 kernel exposes a new API, but still appears to support the old one (for now?).
There's currently code in to handle the new API, but it's commented out.

Long story short, much like in the issue mentioned earlier, I need a strace dump from Nickel, on a Mark7 device running the latest FW, to decide if we need to flip the switch to the new API on those devices ;).

Quoting the relevant instructions:

----

The binaries are available [here](http://files.ak-team.com/niluje/mrpub/Other/strace-kobo.tar.gz).

Basically, you need SSH access *under Nickel*, and with the attached binary, you trace nickel's ioctls, like so: `./strace -fitv -e trace=ioctl -p $(pidof nickel)`. Then you move around a bit, first in the Library, then in a book (where you make sure you do so long enough to trigger at least once a flashing update), moving around in a couple of menus while you're there, and then post the output here ;).

**Comment by zavorra** [comment]
This is the requested log. Let me know if I can help some more.

[clara_hd_nickel_strace.zip](https://github.com/koreader/koreader-base/files/2183623/clara_hd_nickel_strace.zip)

**Comment by NiLuJe** [comment]
That is indeed perfect, thanks :).

Good news: it behaves roughly like an Oasis 2, which is very good news for everyone :).
